AuroraAssetEditor/Classes/XboxAssetDownloader.cs

```diff
@@ -267,7 +267,7 @@ private static void ParseHtml(string url, XboxTitleInfo titleInfo)
             catch {  }
 
             titleInfo.AssetsInfo = ret.ToArray();
-            XboxAssetDownloader.SendStatusChanged("Finished parsing Title/Asset info...");
+            XboxAssetDownloader.SendStatusChanged("Finished fetching assets!");
```

Comment:
This should probably be "Finished fetching online assets..."

Reply:
Fair enough, I can change this as well as the others you suggested.